app.rb

```diff
@@ -0,0 +1,115 @@
+require './book'
+require './classroom'
+require './per_son'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_booklist
+    puts 'There is no book found in the store' if
+    @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def all_peoplelist
+    print 'Missing student or teacher? create student (1) or a teacher (2)
+        [Input the number]: '
+    option = gets.chomp
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'invalid input'
+    end
+    puts 'People created successfully'
+  end
+
+  def create_teacher
+    print 'teacher age: '
+    age = gets.chomp
+
+    print 'teacher specialization'
+    specialization = gets.chomp
+
+    print 'teacher name: '
+    name = gets.chomp
+
+    teacher = Teacher.new(age, specialization, name, parent_permission: true)
+    puts 'New teacher was created succefully'
+    @people.push(teacher)
+  end
+
+  def create_student
+    print 'student age: '
+    age = gets.chomp
+
+    print ' Parent permission [Yes or No]: '
+    parent_permission = gets.chomp.downcase
+    case parent_permission
+    when 'No'
+      student = Student.new(age, name, parent_permission: false)
+      @people.push(student)
+    when 'Yes'
+      student = Student.new(age, name, parent_permission: true)
+      @people.push(student)
+    end
+    puts 'New student was created successfully'
+  end
```

Comment:
- [ ] While creating a student/teacher I am getting this error please address that
![st](https://user-images.githubusercontent.com/44745855/169657858-bc44dd67-ffd1-4ec6-a0e0-a4395da173cb.png)

Reply:
Understood, I have corrected @DeliceLydia